Parent/Parent/Conversations/ConversationDetailCell.swift

```diff
@@ -148,7 +148,7 @@ class ConversationDetailCell: UITableViewCell {
 
             button.pin(inside: self)
 
-            layer.borderColor = tintColor.cgColor
+            layer.borderColor = UIColor.named(.backgroundMedium).cgColor
```

Comment:
I know it's a nitpick, but can you use `borderMedium` instead of `backgroundMedium`?